### PR TITLE
feat: Add CDN example.

### PR DESCRIPTION
**Pull request by wolfgangcodes**
## Which problem is this PR solving?
Published a UMD build for POC. It should be available at 
`https://unpkg.com/@honeycombio/opentelemetry-web/dist/umd/index.js`. Adds an example that serves it locally and exercises the API exposed. 

## Short description of the changes
Published a UMD build for POC

### CODE REVIEW

File: packages/honeycomb-opentelemetry-web/src/cdn.ts
Comment (wolfgangcodes) [inline packages/honeycomb-opentelemetry-web/src/cdn.ts:1]: I'd like to move this to it's own package, but we'll do that in it's own PR.

File: packages/honeycomb-opentelemetry-web/rollup.config.mjs [hunk 1]
```diff
@@ -58,6 +56,46 @@ const esmConfig = {
...
+const cdnConfig = {
+  onwarn(warning, defaultHandler) {
+    if (IGNORE_WARNINGS.includes(warning.code)) {
+      return;
+    }
+    defaultHandler(warning);
+  },
+  input: './src/cdn.ts',
+  output: {
+    file: 'dist/umd/index.js',
```
Comment (wolfgangcodes) [inline packages/honeycomb-opentelemetry-web/rollup.config.mjs:77]: Using unpkg it should be available at `https://unpkg.com/@honeycombio/opentelemetry-web/dist/umd/index.js`

File: packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "hello-world-cjs",
+  "version": "0.0.1",
+  "description": "An example vanilla that uses cjs modules that works with Honeycomb's OpenTelemetry Web Distro",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "build": "cd $npm_config_local_prefix/../../ && npm run build && cd $npm_config_local_prefix &&  mkdir -p ./dist/umd/ && cp -vR $npm_config_local_prefix/../../dist/umd/index.js ./dist/umd",
```
Comment (arriIsHere) [inline packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json:8]: I noticed none of our other examples actually build the parent project (unless you do the docker image). Should we cut out the build part for consistency? Would make the command a lot shorter too.

**Review by arriIsHere — APPROVED** [review]
Everything looks good and when I ran the example locally it worked great! No pressing changes but I do have a Q about building.

File: packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json (hunk 2 above)
Comment (wolfgangcodes) [reply packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json:8]: I agree. I like how this is turn-key 😸  but it does introduce a variant pattern. 😿. 
I def want to standardize how this work so it works in a predictable way. But I need for expressive tooling.How do you feel about me addressing this more wholistically in a follow up PR?

Comment (arriIsHere) [reply packages/honeycomb-opentelemetry-web/examples/hello-world-cdn/package.json:8]: That sounds great! I definitely agree that having to  do the builds individually is not the best.

**Review by pkanal — APPROVED** [review]

**Pull request merged**